openslides_backend/action/actions/motion_comment/create_delete_update.py

```diff
@@ -1,12 +1,69 @@
+from typing import Any, Dict
+
 from ....models.models import MotionComment
+from ....permissions.permissions import Permissions
+from ....shared.exceptions import PermissionDenied
+from ....shared.patterns import Collection, FullQualifiedId
+from ...action import Action
 from ...action_set import ActionSet
+from ...generics.delete import DeleteAction
+from ...generics.update import UpdateAction
 from ...mixins.create_action_with_inferred_meeting import (
-    get_create_action_with_inferred_meeting,
+    CreateActionWithInferredMeeting,
 )
 from ...util.default_schema import DefaultSchema
 from ...util.register import register_action_set
 
 
+class PermissionMixin(Action):
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        super().check_permissions(instance)
+
+        # get section_id and meeting_id, create vs delete/update case.
+        if "section_id" in instance:
+            section_id = instance["section_id"]
+        else:
+            comment = self.datastore.get(
+                FullQualifiedId(self.model.collection, instance["id"]),
+                ["section_id", "meeting_id"],
```

Comment:
`meeting_id` is not needed here, right?

Reply:
Yes, it is no longer needed here. I removed it.